helm/kanister-operator/values.yaml

```diff
@@ -5,6 +5,10 @@ image:
   repository: ghcr.io/kanisterio/controller
   tag: 0.88.0
   pullPolicy: IfNotPresent
+repositoryServerImage:
+  repository: ghcr.io/kanisterio/repo-server-controller
```

Comment:
```suggestion
  registry: ghcr.io/kanisterio
  name: repo-server-controller
```
I think we should break down registry and name into two separate fields. This would allow using a different registry if required.

Reply:
Wouldnt that be inconsistent with current kanister controller image format?